test/unit/spec/v2/ion/uiSchemaTransformer_spec.js

```diff
@@ -1,13 +1,13 @@
 import { _ } from 'okta';
 import responseTransformer from 'v2/ion/responseTransformer';
 import uiSchemaTransformer from 'v2/ion/uiSchemaTransformer';
-import XHRFactorRequiredEmail from '../../../helpers/xhr/v2/FACTOR_REQUIRED_EMAIL';
-import XHREnrollProfile from '../../../helpers/xhr/v2/ENROLL_PROFILE';
-import XHRFactorEnrollOptions from '../../../helpers/xhr/v2/FACTOR_ENROLL_OPTIONS';
+import XHRFactorRequiredEmail  from '../../../../../playground/mocks/idp/idx/data/factor-verification-email.json';
```

Comment:
Add a test cases to cover authenticator cases ?

Reply:
Good catch! I have missed that.